build.gradle

```diff
@@ -67,4 +67,19 @@ subprojects {
       }
     }
   }
+
+  configurations {
```

Comment:
Do we have configurations which we do not want to lock?

```
dependencyLocking {
    lockAllConfigurations()
}
```

might be simpler if not.

Reply:
I tried that first and gradle discovered 20+ configurations, most related to tests, quality tasks, and plugins. We only need to keep an eye on the dependencies we bring to us and users.